### ISSUE TITLE
[CSS2] !important not in index

### ISSUE THREAD

**Issue by gsnedders** [w3c/csswg-drafts#1029]
It seems, uh, important enough that it should be there. :)

**Comment by AmeliaBR** [comment]
It also doesn't show up in https://drafts.csswg.org/indexes/ anywhere.  I'm not sure under _which_ category there it should go ([Grammar productions, maybe?](https://drafts.csswg.org/indexes/#types)), but as with CSS2: it should be somewhere.

**Comment by svgeesus** [comment]
See issue 1 in the index:

> A notable absence at the moment is a list of all the "dfn" type definitions—the ordinary words and phrases given meaning in CSS. This will show up shortly, as soon as I add a mechanism to limit the index generation to specs from a particular group.

So yeah currently we have properties, descriptors, values,  at-rules, functions, selectors.

On the other hand Property/Descriptor Values already lists keywords like inherit. Maybe it should be added there.

cc @tabatkins to comment on whether that needs a markup change, a bikeshed change, or what.

**Comment by tabatkins** [comment]
This isn't an Indexes issue, it's a 2.1 issue - @gsnedders is saying that it doesn't appear in the *2.1 index*. And it doesn't show up there because it's not actually marked up as a definition anywhere in 2.1.  (And then, *because* of that, it obviously doesn't show up in Bikeshed's definition database, and thus not in any indexes.)

As for how to mark it up, I suspect we want to match the global keywords and treat it as a "value" type dfn for 'all'; so in 2.1 it would get `<dfn data-dfn-type=value data-dfn-for=all>!important</dfn>`.

**Comment by AmeliaBR** [comment]
> This isn't an Indexes issue, it's a 2.1 issue

Yep, sorry for distracting from the main issue. The CSS 2.x spec index should have all key term definitions listed, so that should be fixable separate from deciding how to include `!important` in the global index.
